bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -114,38 +123,38 @@ if args.make_signal_kde:
                          (len(mass2), N_original, str(args.min_mass)))
     else:
         idx = numpy.full(N_original, True)
-    mass2_s = mass2
-    mass1_s = data_signal['mass1'][idx]
-    assert min(mass1_s - mass2_s) > 0
-    eta_s = data_signal['eta'][idx]
-    chi_eff_s  = data_signal['chi_eff'][idx]
-    mchirp_s = data_signal['mchirp'][idx]
+    mass1 = data_signal['mass1'][idx]
+    assert min(mass1 - mass2) > 0
+    eta = data_signal['eta'][idx]
+    chi_eff  = data_signal['chi_eff'][idx]
+    mchirp = data_signal['mchirp'][idx]
 
-    param0 = eval(args.fit_param[0][:])
-    param1 = eval(args.fit_param[1][:])
-    param2 = eval(args.fit_param[2][:])
-    if args.log_param == ['False', 'False', 'False']:
-        sample_s = numpy.vstack((param0, param1, param2)).T
-    elif args.log_param == ['True', 'False', 'False']:
-        sample_s = numpy.vstack((numpy.log(param0), param1, param2)).T
-    elif args.log_param == ['True', 'True', 'False']:
-        sample_s = numpy.vstack((numpy.log(param0), numpy.log(param1), param2)).T
-    else:
-        raise ValueError("invalid log_param argument, use 'True', or 'False'")
+    for param, slog in zip(args.fit_param, args.log_param):
+        data = data_signal[param]
+        if slog in ['false', 'False', 'FALSE']:
```

Comment:
Same comments here as for the template param case, you are only allowing the strings 'True' and 'False' as options.